DjangoTemplates/templates/dt1_template/cards/ratings1.html

```diff
@@ -0,0 +1,21 @@
+<div class="card">
+    <div class="card-body">
+        <div class="item-rating-info d-flex mb-4">
+            <h5 class="mr-3">{{ card.title }}</h5>
+            <span class="item-ratings fs-14">
+                {% for icon in card.rating_icons %}
```

Comment:
@TorinFelton could card just accept the number of stars and not a list of icon classes?

Reply:
I was going to do that originally, but then I thought maybe they'd want other icons for the stars potentially? Seems unnecessary in hindsight though, I agree